mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/services/HistorikkService.kt

```diff
@@ -46,19 +45,23 @@ class HistorikkService(
     private fun getHistorikkForBrukerFromDb(norskIdent: String): List<HistorikkForDeltaker> {
         @Language("PostgreSQL")
         val query = """
-            select deltaker.id,
-                   deltaker.fra_dato,
-                   deltaker.til_dato,
-                   deltaker.status,
+            select historikk.id,
+                   historikk.fra_dato,
+                   historikk.til_dato,
+                   historikk.status,
                    gjennomforing.navn,
-                   gjennomforing.virksomhetsnummer,
-                   gjennomforing.tiltaksnummer,
-                   tiltakstype.navn as tiltakstype
-            from deltaker
-                     left join tiltaksgjennomforing gjennomforing on gjennomforing.id = deltaker.tiltaksgjennomforing_id
-                     left join tiltakstype on tiltakstype.id = gjennomforing.tiltakstype_id
+                   gjennomforing.virksomhetsnummer as virksomhetsnummerFraGjennomforing,
+                   tiltakstypeFraGjennomforing.navn as tiltakstypeFraGjennomforing,
+                   tiltakstypeFraTabell.navn as tiltakstypeFraTabell,
+                   historikk.virksomhetsnummer,
+                   historikk.beskrivelse
+            from historikk
+                     left join tiltaksgjennomforing gjennomforing on gjennomforing.id = historikk.tiltaksgjennomforing_id
+                     left join tiltakstype tiltakstypeFraGjennomforing on tiltakstypeFraGjennomforing.id = gjennomforing.tiltakstype_id
+                     left join tiltakstype tiltakstypeFraTabell on tiltakstypeFraTabell.id = historikk.tiltakstypeid 
+                      
             where norsk_ident = ?
```

Comment:
her kan vi også legge på `where norsk_ident = ? and historikk.fra_dato mindre enn eller lik 04.12.2017` så vi bare viser historikk senere enn eller lik 04.12.2017.